RPIPs/RPIP-vop.md

```diff
@@ -0,0 +1,309 @@
+---
+rpip: 
+title: Voted Operational Process Framework
+description: 
+author: LongForWisdom (@LongForWisdom)
+discussions-to: 
+status: Draft
+type: Meta
+created: 2023-10-02
+---
+
+## Abstract
+Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.
+
+$TODO
+
+## Motivation
+
+The motivation for some VOP-like framework can be summarized as the desire for increased transparency and extensibility for DAO operational decision-making.
+
+**Transparency**
+
+A VOP-like framework allows participants in the Rocket Pool DAOs to more easily be able to understand the common processes that shape the DAOs. It does this in a few main ways:
+* The types of process that exist can be listed, summarized and presented in such a way that is more-approachable to the average participant than reading through multiple full RPIPs. Participant attention is a key resource and should not be wasted on implementation details, if all that is desired is a high-level overview of 'how does process x work in Rocket Pool'.
```

Comment:
This is a main point I biffed, I was reading it as "okay something isn't ready for an RPIP but can still be formalized."

Rather the point here is to contain these processes in a more organized way. Dont think that was a fault of the text as much as my preconceived notions from our talks